src/car.rs

```diff
@@ -111,7 +131,7 @@ fn calculate_tire_acceleration_and_braking_forces(
 }
 
 fn turn_tires(keys: Res<Input<KeyCode>>, mut tires: Query<(&mut Transform, &Tire)>) {
-    let turning_radius = PI / 5.0;
+    let turning_radius = 0.296706;
```

Comment:
How'd you choose this number

Reply:
Its the turning radius of the F1 car my roommate is building, so its a realistic number.